pkg/storages/sh/folder.go

```diff
@@ -3,15 +3,13 @@ package sh
 import (
 	"bufio"
 	"errors"
-	"fmt"
 	"io"
 	"os"
 	"path/filepath"
 
-	"github.com/pkg/sftp"
 	"github.com/wal-g/tracelog"
 	"github.com/wal-g/wal-g/pkg/storages/storage"
-	"golang.org/x/crypto/ssh"
+	"github.com/wal-g/wal-g/utility"
```

Comment:
`wal-g/wal-g/pkg/storages` is a standalone package and it should not rely on any other `wal-g/wal-g/*` package.

Reply:
Now SFTP client locates in `wal-g/pkg/sftp`.